cocotb/binary.py

```diff
@@ -69,7 +69,7 @@ class BinaryValue(object):
     _resolve_to_0    = "xXzZuU"  # noqa
     _permitted_chars = "xXzZuU" + "01"  # noqa
 
-    def __init__(self, value=None, bits=None, bigEndian=True,
+    def __init__(self, value=None, bits=None, bigEndian=False,
```

Comment:
@luked80 my fear about changing the default behaviour here is that it will break a lot of existing deployments.

One thing we've mooted in the past is adding a format type to `SimHandleBase` which would control the interpretation of the value (beyond `float` etc for types clearly defined by the RTL).  These could be hierarchical, such that you could globally switch to `littleEndian` by changing the DUT at the toplevel.